tarea 2/Luis Ernesto/fecha.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fecha</title>
+</head>
+<body>
+    <script>
+        (function(){
+
+            let fecha = new Date (); 
+
+            let semana = ["Domingo", "Lunes", "Martes", "Miercoles", "Jueves", "Sabado"]; //arreglo para ajustar el dia de la semana
```

Comment:
Te faltó el Viernes 🙃  y faltan algunos acentos. También, deberías usar `const` siempre que defines un arreglo, sobre todo si no va a cambiar.

```suggestion
            const semana = ["Domingo", "Lunes", "Martes", "Miércoles", "Jueves", "Viernes", "Sábado"]; //arreglo para ajustar el día de la semana
```